kinesify-data.js

```diff
@@ -54,13 +54,8 @@ function fixItem (item) {
 
 function fixBib (bib) {
   bib.varFields = (bib.varFields || []).map((field) => {
-    // Seems that var subFields sometimes appear under `subfields` in violation of schema, so correct it:
```

Comment:
@jobinthomasnypl This your change too?

Reply:
Yes, except that I was setting it to null too which seems like you are removing.

Reply:
Actually, yeah, maybe I should have been a bit more thorough in what I remove here...

Reply:
Yes, it's not needed anywhere — removing is good.

That said, all the test data still use `subField`s, so I think there's a wider cleanup task. But this will get `kinesify-data.js` working again at least.

Reply:
Was there a recent change made to the Bib/Item avro schema - changing`subFields` to `subfields`? What's the correct case?

Reply:
Yeah the correct case is `subfields` and `subfield` - all lower case. That is how sierra returns its response. We were changing it to subFields and that was causing issues before our 1.0 release. The schema in prod reflects this change though.

Reply:
Thanks, @jobinthomasnypl - Curious: Why were we changing `subfields` it to `subFields` previously?